src/main/resources/mappings/global-context.json

```diff
@@ -0,0 +1,69 @@
+{
+  "dynamic": false,
+  "_meta": {
+    "schema_version": 1
+  },
+  "properties": {
+    "workflow_id": {
+      "type": "keyword"
+    },
+    "name": {
+      "type": "text",
+      "fields": {
+        "keyword": {
+          "type": "keyword",
+          "ignore_above": 256
+        }
+      }
+    },
+    "description": {
+      "type": "text"
+    },
+    "use_case": {
+      "type": "keyword"
+    },
+    "operations": {
+      "type": "keyword"
+    },
+    "version": {
+      "type": "nested",
+      "properties": {
+        "template": {
+          "type": "integer"
+        },
+        "compatibility": {
+          "type": "integer"
+        }
+      }
+    },
+    "user_inputs": {
+      "type": "nested",
+      "properties": {
+        "model_id": {
```

Comment:
Should we remove these specific fields, we might not want to have fields here that are specific to a use case, and make the fields for the GC mapping completely use-case independent

Reply:
`model_id` would be generated after we register a model and won't be a user input. We can remove this.

Reply:
I think we should remove all fields that are specific to a use case from the GC mapping. We can't sustainably add more and more fields as we have more and more use cases

Reply:
Updated `user_inputs` mapping like this to keep it align with the final iteration of use case template. Will mention this in the global context followup issue to see if we want to further update the mapping to make it a generic text field

```
    "user_inputs": {
      "type": "nested",
      "properties": {
        "index_name": {
          "type": "keyword"
        },
        "index_setting": {
          "type": "keyword"
        }
      }
    },
```